draft-ietf-rats-corim.md

```diff
@@ -995,19 +990,25 @@ the sensitive values in memory are encrypted.
 Raw value measurements are typically vendor defined values that are checked by Verifiers
 for consistency only, since the security relevance is opaque to Verifiers.
 
-There are two parts to a `raw-value-group`, a measurement and an optional mask.
+A raw value from Evidence, or an Endorsement, is a value of type `bytes` with an optional tag value.
 The default raw value measurement is of type `tagged-bytes` ({{sec-common-tagged-bytes}}).
-Additional raw value types can be defined, but must be CBOR tagged so that parsers can distinguish
-between the various semantics of type values.
 
+Additional raw value types can be defined, but must be a CBOR tagged `bstr`.
+This allows parsers to distinguish between the various semantics of raw values, and lets Verifiers compare raw values without understanding their contents.
+
+A raw value intended for comparison, for example in `stateful-environment-record` can include a mask value, which selects the bits to compare during appraisal.
 The mask is applied by the Verifier as part of appraisal.
 Only the raw value bits with corresponding TRUE mask bits are compared during appraisal.
 
-When a new raw value type is defined, the convention for applying the mask is also defined.
-Typically, a CoRIM profile is used to define new raw values and mask semantics.
+When a new raw value type is defined, a comparison algorithm may also be defined.
```

Comment:
Nooo. This should be a measurement value map extensionnn

Reply:
Trying to keep the spirit of what was there before. Happy to lose this sentence.